src/components/Button/Button.tsx

```diff
@@ -78,7 +78,7 @@ export const Button: React.FC<Props> = ({
   buttonRef,
   className,
   variant,
-  size,
+  size = 'lg',
```

Comment:
_technically_ in the docs now it suggests `size` should always default to `'md'`. we've talked about this at length before but maybe can have design take another look at the variants and maybe remove one of the smaller sizes 🥲

Reply:
>technically in the docs now it suggests size should always default to 'md'

I dunno, that feels weirdly restrictive to me and I'd personally rather change the docs than force the button sizes to comply. But if we really wanted to make it work, another solution would be to bump the buttons sizes down a level ('large' -> 'md', 'medium' -> 'sm', 'small' -> 'xs').

Reply:
But I don't want to add a new large size if we have no actual intention of using it.

Reply:
yeah, fair. Changing the docs sounds fine